manifests/cluster-federation/0.0.5/cluster-federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,201 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: cluster-federation.v0.0.5
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: "OpenShift Optional, Integration & Delivery"
+    description: Cluster-scoped Federation
+    certified: "false"
+    containerImage: "quay.io/openshift/origin-federation-controller:v4.0.0"
+    createdAt: "2019-01-01T00:00:00Z"
+spec:
+  displayName: Cluster Federation
+  description: |
```

Comment:
@robszumski can you help figure out how to meaningfully differentiate this from the other CSV?